Automattic/wp-calypso
# Add missing features to the new Jetpack plans

### Changes proposed in this Pull Request

In this PR, we've made sure that new Jetpack plans included legacy plans features that were still relevant.

### Implementation notes

- I've tried to match the table of features mentioned in the testing instructions below
- I didn't include features that are available for all plans

### Testing instructions

- Get the list features that's linked in the _Project Quick References_ card
- Check that the added features make sense
